src/services/rates/currencyConverter.test.ts

```
@@ -0,0 +1,312 @@
+    });
+
+    it("should reject unsupported currency", async () => {
+      await expect(convertLocalToUsd(100000, "JPY")).rejects.toThrow(
+        expect.objectContaining({
+          message: expect.stringContaining("Currency JPY not supported"),
+          statusCode: 400,
+        }),
+      );
+    });
+
+    it("should handle case-insensitive currency codes", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)